x-pack/plugins/security_solution/public/common/components/line_clamp/index.tsx

```diff
@@ -91,7 +95,11 @@ const LineClampComponent: React.FC<{
         children
       )}
       {isOverflow && (
-        <ReadMore onClick={toggleReadMore} size="s" data-test-subj="summary-view-readmore">
+        <ReadMore
+          onClick={onReadMore ?? toggleReadMore}
```

Comment:
Could we create another link component here for navigating to notes tab and leave the ReadMore as it is?

Reply:
It makes sense to navigate from qeury tab to notes tab with the read more button, but after landing on the notes tab, the read more link is still there but not doing anything atm.

Reply:
In the notes tab, it only scrolls to the description note. Should we hide the "read more" button if the selected tab is "Notes"? 🤔

Reply:
To me, I would expect if the button and the cropped description is there, it would still expand the content, but I have no strong opinion.
I think @monina-n would have better idea of how to deal with this :D